dao/src/main/resources/sql-query.properties

```diff
@@ -70,3 +70,16 @@ car.sale.add=INSERT INTO car_sale (price, user_id, car_id) VALUES (:price, :user
 car.sale.update=UPDATE car_sale SET price = :price, user_id = :userId, car_id = :carId \
   WHERE car_sale_id = :id
 car.sale.delete=DELETE FROM car_sale WHERE car_sale_id = :id
+#CAR_CHARACTERISTICS
+car.characteristics.get=SELECT CONCAT('{',\
+  '"carMakeList":[',group_concat(DISTINCT concat('{"carMakeId":',cm.car_make_id,\
+  ',"carMake":"', cm.car_make,'"}') separator ','),'],',\
+  '"fuelTypeList":[',group_concat(DISTINCT concat('{"fuelTypeId":',ft.fuel_type_id,\
+  ',"fuelType":"', ft.fuel_type,'"}') separator ','),'],',\
+  '"transmissionList":[',group_concat(DISTINCT concat('{"transmissionId":',tr.transmission_id,\
+  ',"transmission":"', tr.transmission,'"}') separator ','),'],',\
+  '"carFeatureList":[',group_concat(DISTINCT concat('{"carFeatureId":',cf.car_feature_id,\
+  ',"carFeature":"', cf.car_feature,'"}') separator ','),']',\
+  '}') as car_characteristics \
+  FROM car_make as cm, fuel_type as ft, transmission as tr,\
+  car_feature as cf;
```

Comment:
https://database.guide/json_object-create-a-json-object-from-a-list-of-key-value-pairs-in-mysql/